impeller/entity/contents/test/contents_test_helpers.h

```diff
@@ -11,26 +11,34 @@ namespace impeller {
 /// @brief Retrieve the [VertInfo] struct data from the provided [command].
 template <typename T>
 typename T::VertInfo* GetVertInfo(const Command& command) {
-  auto resource = command.vertex_bindings.buffers.find(0u);
+  auto resource = std::find_if(command.vertex_bindings.buffers.begin(),
+                               command.vertex_bindings.buffers.end(),
+                               [](const BufferAndUniformSlot& data) {
+                                 return data.slot.ext_res_0 == 0u;
+                               });
   if (resource == command.vertex_bindings.buffers.end()) {
     return nullptr;
   }
 
-  auto data = (resource->second.view.resource.contents +
-               resource->second.view.resource.range.offset);
+  auto data =
+      (resource->view.resource.contents + resource->view.resource.range.offset);
   return reinterpret_cast<typename T::VertInfo*>(data);
 }
 
 /// @brief Retrieve the [FragInfo] struct data from the provided [command].
 template <typename T>
 typename T::FragInfo* GetFragInfo(const Command& command) {
-  auto resource = command.fragment_bindings.buffers.find(0u);
+  auto resource = std::find_if(command.fragment_bindings.buffers.begin(),
```

Comment:
So we are getting linear search here hopefully?  This should be faster for the small sizes we have.

Reply:
This is just for testing, so performance doesnt matter as much

Reply:
We dont ever have to look up particular bindings, we just iterate through the set of bindings and bind each one.